src/Platform/Microsoft.Testing.Platform/Helpers/System/IChannel.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#if NETCOREAPP
+namespace Microsoft.Testing.Platform.Helpers;
+
+internal interface IChannel<T>
```

Comment:
why we need to mock this types? I would not do it to not pay virtual dispatch on these core hot loop objects?

Reply:
Same as below.